foodalert/static/foodalert/js/base.js

```diff
@@ -13,6 +13,8 @@ window.vueData.components = {
 };
 
 window.vueData.routes = [
+  { path: "/welcome", component: UnauthzTemplate},
```

Comment:
Is the idea that we want this to be the first page all users hit? If so, it should still probably be a child route our Generic Template, but we may need to discuss the flow/ how this page is meant to be visited in person.

Reply:
I talked to Adi about this and we put it outside the Generic Template because it got rid of the empty heading bar at the top.